spec/exchanges/allcoin/integration/market_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+RSpec.describe 'Allcoin integration specs' do
+  let(:client) { Cryptoexchange::Client.new }
+
+  it 'fetch pairs' do
+    pairs = client.pairs('allcoin')
+    expect(pairs).not_to be_empty
+
+    pair = pairs.first
+    expect(pair.base).to_not be nil
+    expect(pair.target).to_not be nil
+    expect(pair.market).to eq 'allcoin'
+  end
+
+  it 'fetch ticker' do
+    btc_usd_pair = Cryptoexchange::Models::MarketPair.new(base: 'btc', target: 'usd', market: 'allcoin')
+    ticker = client.ticker(btc_usd_pair)
+
+    expect(ticker.base).to eq 'BTC'
+    expect(ticker.target).to eq 'USD'
+    expect(ticker.market).to eq 'allcoin'
+    expect(ticker.last).to be_a Numeric
+    expect(ticker.ask).to be_a Numeric
+    expect(ticker.bid).to be_a Numeric
+    expect(ticker.volume).to be_a Numeric
+    expect(ticker.timestamp).to be_a Numeric
+    expect(DateTime.strptime(ticker.timestamp.to_s, '%s').year).to eq Date.today.year
```

Comment:
@yihyang this will fail after 2017, right?

Reply:
@dominicwong617 just curious why will it fail?

the timestamp is retrieved from the following line:
https://github.com/yihyang/cryptoexchange/blob/b37763bab81cde0e6be6969c7e4b42e995311054/lib/cryptoexchange/exchanges/allcoin/services/market.rb#L34

Which is provided by the allcoin's API in here:
https://allcoin.com/About/APIReference/

Reply:
Because VCR locks the timestamp to be 2017, and Date.today.year will return 2018 once it is 2018?

Reply:
That's true and it may apply to some of the other specs too now that we are using VCR
How about checking if it falls within a reasonable range. Otherwise we get funny errors that we are not aware that we need to update VCR cassettes.

Reply:
ok noted @tmlee @dominicwong617 thanks!